### PR TITLE
Utility refactoring #224

### PR DESCRIPTION
**Pull request by vDrWool**

### CODE REVIEW

**Review by Svidunovich — APPROVED** [review]

File: Utility/Utility.Public/Include/Utility/WarningSuppression.hpp [hunk 1]
```diff
@@ -7,11 +7,11 @@
...
-													  _doPragma(GCC diagnostic ignored #GCC_CLANG_TEXT)
+													  _doPragma(GCC diagnostic ignored GCC_CLANG_TEXT)
 #	define restoreWarning _doPragma(GCC diagnostic pop)
 #elif __clang__
 #	define _doPragma(x) _Pragma(#x)
 #	define suppressWarning(MSVC_CODE, GCC_CLANG_TEXT) _doPragma(clang diagnostic push) \
-													  _doPragma(clang diagnostic ignored #GCC_CLANG_TEXT)
+													  _doPragma(clang diagnostic ignored GCC_CLANG_TEXT)
 #	define restoreWarning _doPragma(clang diagnostic pop)
 #endif
```
Comment (HRenata) [inline Utility/Utility.Public/Include/Utility/WarningSuppression.hpp:17]: new line here

File: Client/Client.CLI/App.cpp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #include "App.hpp"
-#include "Utility/YasSerializer.hpp"
+#include "Network/YasSerializer.hpp"
```
Comment (HRenata) [inline Client/Client.CLI/App.cpp:2]: Why did you move serializatio, compression, encryption handlers from utility to network?

**Review by HRenata — CHANGES_REQUESTED** [review]
W

File: Client/Client.CLI/App.cpp (hunk 2 above)
Comment (vDrWool) [reply Client/Client.CLI/App.cpp:2]: Because it's using namespace Network.

**Review by StasDash — APPROVED** [review]

**Review by f1nal3 — APPROVED** [review]

**Comment by f1nal3** [comment]
Pls update n merge
